### ISSUE TITLE
Error: The "RunCommand" property is not defined

### ISSUE THREAD

**Issue by JipingWang** [aspnet/Blazor#952]
Run a new project from the template getting error:

> Unable to run your project. The "RunCommand" property is not defined.

```
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>netstandard2.0</TargetFramework>
    <OutputType>Exe</OutputType>
    <LangVersion>7.3</LangVersion>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.Blazor.Browser" Version="0.4.0" />
    <PackageReference Include="Microsoft.AspNetCore.Blazor.Build" Version="0.4.0" />
  </ItemGroup>

  <ItemGroup>
    <ProjectReference Include="..\Blazor040_2.Shared\Blazor040_2.Shared.csproj" />
  </ItemGroup>

</Project>
```

**Comment by JipingWang** [comment]
Another computer is working fine. But this one reports:
![blazor-error](https://user-images.githubusercontent.com/14056637/41131956-2b6e0c4c-6a8c-11e8-9217-0eb44b8acd0b.PNG)

**Comment by JipingWang** [comment]
Solved referencing to #716 .

**Comment by SteveSandersonMS** [comment]
Glad you solved it!
